src/assets/translations/fr/main.json

```diff
@@ -2131,17 +2137,24 @@
       "repaymentLock": "Délai minimum avant de pouvoir rembourser votre prêt",
       "collateralizationRatio": "Rapport entre la valeur de votre collatéral et le montant du prêt",
       "feesPlusSlippage": "Total des frais perçus, y compris les frais L1, le glissement/slippage et les frais de liquidité de THORChain"
+    },
+    "yourLoans": {
+      "yourLoans": "Vos prêts",
+      "emptyTitle": "Le pays d'Aucun Prêt !",
```

Comment:
hahaha

Reply:
feel free to get creative if you want to get some funny one here as well!

Reply:
I wanted to go with "Zero" something too, but "Prêt zéro" sounded too much like these 0% internet loans 😆 
Thanks for the review !